Makefile

```
@@ -21,7 +21,12 @@ IMAGE_PREFIX                := $(REGISTRY)/extensions
 REPO_ROOT                   := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 HACK_DIR                    := $(REPO_ROOT)/hack
 VERSION                     := $(shell cat "$(REPO_ROOT)/VERSION")
-LD_FLAGS                    := $(shell ./vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh github.com/gardener/"$(EXTENSION_PREFIX)-$(NAME)/pkg" "$(REPO_ROOT)/VERSION")
+
+# The project currently includes two version tables, so generate LD_FLAGS for both
+LD_FLAGS_GENERATOR			:= $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh
```

You could use spaces over tabs:
```suggestion
LD_FLAGS_GENERATOR          := $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh
```